### DISCUSSION TITLE
Weighted omega mean

### DISCUSSION THREAD

**Discussion by jjepastor** [mtex-toolbox/mtex#1584] [Ask Anything]
Hello Ralf and Rudiger!

 Is it posible to compute the weighted mean of the omega angle? What I mean, is considering the size of the fitted ellipse.

Best regards

Pastor Juan

**Comment by kilir** [comment]
Hi, 
one solution would be to turn them into vector3d and use mean() which has an option for weights.
Cheers,

Rüdiger
